blockchain/headercmt.go

```diff
@@ -128,7 +128,7 @@ func (b *BlockChain) FetchUtxoViewParentTemplate(block *wire.MsgBlock) (*UtxoVie
 // when it exists.  This function returns the filters regardless of whether or
 // not their associated block is part of the main chain.
 //
-// An error of type NoFilterError will be returned when the filter for the given
+// An error of kind ErrNoFilter will be returned when the filter for the given
```

Comment:
Maybe say that it wraps ErrNoFilter?  That would make it more obvious that errors.Is/As can be used for detection instead of implying that one may type assert to this error kind (which won't work, as the concrete type is ContextError).

Reply:
Fair point.  Updated.